avm/res/kubernetes-configuration/extension/main.json

```diff
@@ -100,7 +100,7 @@
       "condition": "[parameters('enableTelemetry')]",
       "type": "Microsoft.Resources/deployments",
       "apiVersion": "2023-07-01",
-      "name": "[format('46d3xbcp.res.kubernetesconfiguration-fluxconfig.{0}.{1}', replace('-..--..-', '.', '-'), substring(uniqueString(deployment().name, parameters('location')), 0, 4))]",
+      "name": "[format('46d3xbcp.res.kubernetesconfiguration-extension.{0}.{1}', replace('-..--..-', '.', '-'), substring(uniqueString(deployment().name, parameters('location')), 0, 4))]",
```

Comment:
I'd suggest to:
- merge the telemetry fix with the kube flux PR so that the latest version is correct
- Add to this PR the reference to the latest kube flux in the kube extension module

Otherwise we'll release 2 versions with wrong telemetry

Reply:
Hm ok. Otherweise we could've just merged this one first

Reply:
Addressed: https://github.com/Azure/bicep-registry-modules/pull/587

Reply:
Missing the extension updates we removed from #625
We should reference the latest kube flux version and related parameters. We should also bump the kube extension minor version

Reply:
Awaiting #641

Reply:
#641 is merged

Reply:
Updated and now retesting: https://github.com/AlexanderSehr/bicep-registry-modules/actions/runs/6896024296